doc/checkstyle.xsd

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2016 LinkedIn Corp.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
are we allow to republish this under MIT here?

Reply:
I'm not sure.

We have to keep this file licensed under the Apache 2.0 license but we may have the right to dual license it.

Reply:
I would keep it `as is`, just add meta information about it with a link to origin